app/assets/stylesheets/code_example.scss

```
@@ -0,0 +1,22 @@
+@import "constants";
+@import "colors";
+
+.code-example {
+  background-color: var(--background-contrast-grey);
+
+  .example-header {
+    display: flex;
+    background-color: $light-grey;
```

```suggestion
    background-color: var(--background-alt-grey); 
```
Pour le thème sombre, en général il ne faut plus utiliser les variables SASS